plenum/server/replica_validator.py

```diff
@@ -107,9 +107,13 @@ def can_send_3pc_batch(self):
             return False
         if self.replica.viewNo < self.replica.last_ordered_3pc[0]:
             return False
-        if self.replica.viewNo == self.replica.last_ordered_3pc[0] and \
-                self.replica.lastPrePrepareSeqNo < self.replica.last_ordered_3pc[1]:
-            return False
+        if self.replica.viewNo == self.replica.last_ordered_3pc[0]:
+            if self.replica.lastPrePrepareSeqNo < self.replica.last_ordered_3pc[1]:
+                return False
+            if self.replica.config.Max3PCBatchesInFlight is not None:
```

Comment:
Please add a comment why this check is inside `self.replica.viewNo == self.replica.last_ordered_3pc[0]` (as I understand, this is to avoid issues after view change to send enough batches when we have ppSeqNo=1).